src/routes/add_user.js

```diff
@@ -0,0 +1,21 @@
+const MongoClient = require('mongodb').MongoClient;
+const dbQueries = require('../db_queries.js');
+const bcrypt = require('bcrypt');
+
+module.exports = (req, res) => {
+  MongoClient.connect(process.env.DATABASE_URL, (err, db) => {
```

Comment:
@mantagen Not sure we understand this. If it's required in multiple places (to access the various query functions) won't it open multiple connections?